<?xml version="1.0"?>

<launch>
	<arg name="robot_node" default="false" />
	
	<!--
	<node pkg="xv_11_laser_driver" type="neato_laser_publisher" name="xv_11_node">
		<param name="port" value="/dev/ttyACM0"/>
		<param name="firmware_version" value="2"/>
		<param name="frame_id" value="laser"/>
	</node>

	<node pkg="tf" type="static_transform_publisher" name="base_frame_2_laser" args="0 0 0 3.142 0 0 /base_link /laser 100"/> 
	<include file="$(find robot_controller)/launch/default_mapping.launch"/> 
	<include file="$(find hector_geotiff)/launch/geotiff_mapper.launch"/>  
	--> 

	<node pkg="rviz" type="rviz" name="rviz" args="-d rviz_cfg_5_7_4_3.rviz" />
        <!-- <remap from="initialpose" to="robot4/initialpose" /> This can be done in rviz "Panels/Tools Properties/2D Pose Estimate/Topic"-->
	
	<!--
	<node pkg="robot_controller" type="robot" name="robot_node" if="$(arg robot_node)" >
		<param name ="trajectory_file" value = "$(find robot_controller)/launch/Orbital_ECI.txt"/>
		<param name ="robot_name" value = "robot1"/>
	</node>
	
	<arg name="record_msgs" default="true" />
	
	<node pkg="rosbag" type="record" name="rosbag_record_msgs"
	   args="-a "
	   if="$(arg record_msgs)" />
	-->
</launch>
